packages/flame/lib/src/components/position_component.dart

```diff
@@ -411,8 +417,9 @@ class PositionComponent extends Component
     final hasSelfYFlip =
         !parentAbsoluteScale.y.isNegative && scale.y.isNegative;
 
-    return (hasOddFlips ? -1 : 1) * angleDifference +
+    final result = (hasOddFlips ? -1 : 1) * angleDifference +
         (hasSelfYFlip ? 1 : 0) * math.pi;
+    return result % tau;
```

Comment:
I'd say we should normalize this between [-pi, pi], since then the user knows which way to rotate the shortest way without having to do anything more.

Reply:
ah yes good call. I originally had that but it was failing tests, so I changed to 0,tau. but that also failed the tests. I am not sure what the test authors were expecting exactly. eventually I gave up and just add he same normalize to the test
I think there is a bigger q of do we normalize angles on flame, where, and do we have a standard. we could export a common util for now, but I think we should start doing it on angle setters for example

Reply:
>  I think there is a bigger q of do we normalize angles on flame, where, and do we have a standard. we could export a common util for now, but I think we should start doing it on angle setters for example

I think this place is the most important one, so we could start here in this PR and then we could make a list of other places that we'd like to normalize and do that in a follow-up?

Reply:
Done in c8a30b2, will add an extra note about it in the release notes.

Reply:
I was a bit skittish of doing it and left a TODO instead, but let's do it! should've been done since the beginning
I suspect it could potentially break some super edge case handling but nothing that couldn't be very easily fixed by the user (also I can't come up with a specific example rn)